### DISCUSSION TITLE
Is there an example of a global filter not using fuzzy matching?

### DISCUSSION THREAD

**Discussion by mdodge-ecgrow** [TanStack/table#4823] [Q&A]
I am using v8 with React.  I just want to have a global filter in my app that does not utilize fuzzy matching.  All I can find in the docs is fuzzy matching and I can't figure out how to remove that option.  Is there a good example someone knows of that has a global filter without fuzzy matching?

**Comment by Hephaestian** [comment]
So the [React Filters example](https://tanstack.com/table/v8/docs/framework/react/examples/filters) includes a custom "fuzzy matching" filter as a way of also demonstrating the power/extensibility of filters. As a demonstration it is interesting to note these advanced capabilities... but it is more than required for a more basic implementation.

For many simpler cases, you can use one of the more basic built-in filter functions to achieve what you need, simply by including the name of that filter function in the Table config (eg. `globalFilterFn: 'equalsString'` ) instead of a reference to your own custom filter function. By way of comparison, you can see a built-in global filter being used in the [Solid Filters example](https://tanstack.com/table/v8/docs/framework/solid/examples/filters).
